tests/jeos/imgsize.pm

```diff
@@ -0,0 +1,23 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run {
+    my $self   = shift;
+    my $iso    = get_var("HDD_1");
+    my $size   = -s $iso;
```

Comment:
Yes, thats the plan..
On 8 Aug 2015 09:09, "Stephan Kulow" <notifications@github.com> wrote:

> In tests/jeos/imgsize.pm
> <https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/641#discussion_r36575869>
> :
>
> > @@ -0,0 +1,23 @@
> > +use base "opensusebasetest";
> > +use strict;
> > +use testapi;
> > +
> > +sub run {
> > +    my $self   = shift;
> > +    my $iso    = get_var("HDD_1");
> > +    my $size   = -s $iso;
>
> the worker expands this before calling os-autoinst
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/641/files#r36575869>
> .
>